pytest_logger/plugin.py

```diff
@@ -14,6 +15,16 @@
     string_type = str
 
 
+@pytest.hookimpl
```

Comment:
Not sure, I checked how logbook plugin implements hooks and this is used there (https://bitbucket.org/flub/pytest-logbook/src/e9881e3b59dfd5ed4179e0a07a52a470cf53d415/pytest_logbook.py?at=default&fileviewer=file-view-default) anyway if it is not mandatory to mark hooks and if it works, can be removed